test/verify/check-system-info

```diff
@@ -1161,5 +1101,76 @@ class TestSystemInfoTime(packagelib.PackageCase):
         b.wait_not_present("#change_systime button:contains('Automatically using additional NTP servers')")
 
 
+class TestSystemInfoSubscribed(submanlib.SubscriptionCase):
+    provision = {
+        "0": {"address": "10.111.112.1/20", "dns": "10.111.112.1", "memory_mb": 1024},
+        "services": {"image": "services", "memory_mb": 1024}
+    }
+
+    @testlib.onlyImage("insights-client is only on RHEL", "rhel*")
+    @testlib.skipBeiboot("no local overrides/config in beiboot mode")
+    def testInsightsStatus(self):
+        m = self.machine
+        b = self.browser
+
+        # insights-client requires a valid subscription
+        self.setup_candlepin_service(self.machines['services'])
+        self.register_with_candlepin()
+
+        # insights-client might get started during boot and might then
+        # run concurrently with our explicit "insights-client
+        # --register" below.  insights-client is not designed to be
+        # run concurrently and there is no protection against it,
+        # apparently.  So let's prevent that.
+        m.execute("systemctl disable --now insights-client")
+
+        self.restore_dir("/etc/insights-client")
+        self.restore_dir("/etc/yum")
+        self.restore_dir("/var/lib/insights")
```

Comment:
So with `provision` this can't be a non-destructive test anymore so this should not be required. And might even raise an exception?

Reply:
Reading `restore_file` it just skips it so it is a "noop" so lets drop it.

Reply:
Right, thanks!